pkg/storageincentives/agent.go

```diff
@@ -25,7 +28,10 @@ import (
 	"github.com/ethersphere/bee/pkg/swarm"
 )
 
-const loggerName = "storageincentives"
+const (
+	loggerName              = "storageincentives"
```

Comment:
Grouping constants and variables is recommended if they are somehow related, which is not the case here.